simulation/traffic_simulator/src/api/api.cpp

```diff
@@ -130,7 +130,6 @@ bool API::spawn(
   simulation_interface::toProto(params, *req.mutable_parameters());
   spawn_misc_object_entity_client_.call(req, res);
   return res.result().success();
-  return true;
```

Comment:
If the entity was failed to spawn in sensor simulator, it res.result().success() == false.
So, please remain this line.

```
return res.result().success();
```

Reply:
Oh! Sorry.
I missed.
Your code works right.